src/adagrams.js

```diff
@@ -1,15 +1,103 @@
+const HAND_SIZE = 10;
+const LETTER_POOL = {
+  A: 9, B: 2, C: 2, D: 4, E: 12, F: 2,
+  G: 3, H: 2, I: 9, J: 1, K: 1, L: 4,
+  M: 2, N: 6, O: 8, P: 2, Q: 1, R: 6,
+  S: 4, T: 6, U: 4, V: 2, W: 2, X: 1,
+  Y: 2, Z: 1
+};
+const SCORE_CHART = {
+  A: 1, E: 1, I: 1, O: 1, U: 1, L: 1, 
+  N: 1, R: 1, S: 1, T: 1, D: 2, G: 2,
+  B: 3, C: 3, M: 3, P: 3, F: 4, H: 4,
+  V: 4, W: 4, Y: 4, K: 5, J: 8, X: 8,
+  Q: 10, Z: 10
+};
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  let letters = []
+  let total_pool = []
```

Comment:
Since you aren't re-assigning these variables and only modifying the contents of the arrays (pushing letters to them), then you can use `const` here instead of `let`. Here is a general guidelines for when to use one vs. the other: 
- Use `const` when you do not plan to reassign the variable. This is ideal for constants, fixed references to objects or arrays, and loop variables that don’t change inside the loop body.

- Use `let` when you need to reassign the variable later, such as in loops with counters, conditionally reassigned values.
- ```suggestion
  const letters = []
  const total_pool = []
```

Reply:
Got it! My thought when I was choosing before was that we had to assign it as `let` since we are reassigning values inside of the arrays, but that makes sense that since the reference itself to the array is not changing, we can use `const`!